dotcom-rendering/src/model/article-schema.json

```diff
@@ -4317,6 +4317,9 @@
                 },
                 "showQuotedHeadline": {
                     "type": "boolean"
+                },
+                "showMainVideo": {
```

Comment:
I can't see where this is set or used - maybe a copy/paste error?

Reply:
It's passed to the `ImageWrapper` component, where we decide whether or not to include the the `PlayIcon` component